UnitTests/GitCommands.Tests/Helpers/PathUtilTest.cs

```diff
@@ -229,5 +229,34 @@ public void FindAncestors(string path, string[] expected)
         {
             Assert.AreEqual(expected, PathUtil.FindAncestors(path).ToArray());
         }
+
+        private static IEnumerable<TestCaseData> InvalidFolders
+        {
+            get
+            {
+                yield return new TestCaseData(null);
+                yield return new TestCaseData("");
+                yield return new TestCaseData(@"A:\does\not\exist");
```

Comment:
😉 I assign `B:` to my backup drives. Some day, I will have a good usage for `A:`, too, and then I'll create these folders.